taxonomy/tasks.py

```diff
@@ -23,8 +23,9 @@ def update_course_skills(course_uuids):
     """
     LOGGER.info('[TAXONOMY] refresh_course_skills task triggered')
     courses = get_course_metadata_provider().get_courses(course_ids=course_uuids)
+    product_type = 'Course'
```

Comment:
since this method is not called from programs in any way, the string 'Course' can be mentioned in argument directly instead of having a variable.

Reply:
hmm, i think passing arguments via variables is more acceptable compared to hard-coding, but it may debatable!

Reply:
Only if the variable is used more than once. There is only one instance of usage here.